### ISSUE TITLE
Teams theme: emoji icon is not centered

### ISSUE THREAD

**Issue by miroslavstastny** [microsoft/fluent-ui-react#715]
### Steps
```jsx
import React from 'react'
import { Provider, Icon } from '@stardust-ui/react'

const IconEx = () => (
  <Provider theme={{
    componentStyles: {
      Icon: {
        root: {
          outline: '1px solid red',
        },
      },
    },
  }}>
    <>
      <Icon name='emoji' />
      <Icon name='emoji' variables={{outline: true}}/>
    </>
  </Provider>
)

export default IconEx
```

### Expected Result
All icons take full space of the element or are centered

### Actual Result
Outline version is correctly centered, filled version is not  💣
![image](https://user-images.githubusercontent.com/9615899/51115652-28375d00-1809-11e9-928a-24a948112452.png)

### Version
0.16.1

**Comment by miroslavstastny** [comment]
@codepretty - could you please check this?

**Comment by alinais** [comment]
resolved #991
